wiki/markdown/configuration.md

```diff
@@ -17,15 +17,13 @@ In this case we use the same instance of RabbitMQ and MongoDB.
   - **rabbitmq.consumerName** property should be different for each rule set. Otherwise the rabbitMQ will split the events in the queue among all the instances listening to that queue.
   - MongoDb collection names should also be different for each rule set.
 
-  <img src="images/multiple_EI_instances.png">
-</img>
+  <img src="images/multiple_EI_instances.png"></img>
```

Comment:
In the top paragraph on this page, you could add that all of these properties can be set either using application.properties or with -D flags when starting up Eiffel Intelligence.